developer_docs/how_to_create_a_plugin.asciidoc

```diff
@@ -1649,6 +1649,14 @@ end
 ----
 
 
+[[extending-hostgroup-ui]]
+=== Extending hostgroup UI
+A plugin can add items displayed in the `Actions` dropdown on hostgroup overview page.
+
+Adding an item to the actions dropdown requires adding a helper with a method named `hostgroups_dropdown_actions` that accepts a hostgroup as an argument
+and returns an array of relevant actions to your plugin.
```

Comment:
can you give an example of how you extended it from foreman_ansible? https://github.com/theforeman/foreman_ansible/pull/541

just the code example to extend the `hostgroups_dropdown_actions` helper

Reply:
I added an example, let me know if it is ok, or if I should change it.